doc/api/crypto.md

```diff
@@ -2070,6 +2073,23 @@ const hashes = crypto.getHashes();
 console.log(hashes); // ['DSA', 'DSA-SHA', 'DSA-SHA1', ...]
 ```
 
+### crypto.isKeyObject(value)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `value` {any}
+* Returns: {boolean}
+
+Returns `true` if the value is a [`KeyObject`] instance.
```

Comment:
I believe we usually use `[...][]`, but I am not sure what the difference is. @vsemozhetbyt might know?

Reply:
I took inspiration from the other references to KeyObject within this doc itself

Reply:
`[...][]` is a common format, `[...]` is a contraction that parsers accept and our docs use not very often. Markdown docs [mention](https://daringfireball.net/projects/markdown/syntax#link) only the full form and this `[...][]` full form is used in our sources most often, so it is preferable.

Reply:
done and fixed the other occurences i took as reference